python/tach/parsing/config.py

```diff
@@ -24,7 +27,14 @@ def dump_project_config_to_yaml(config: ProjectConfig) -> str:
     # show excluded paths.
     config.exclude = list(set(config.exclude)) if config.exclude else []
     config.exclude.sort()
-    return yaml.dump(config.model_dump(exclude_unset=True), sort_keys=False)
+    language_server_directive = "# yaml-language-server: $schema=docs/assets/tach-yml-schema.json\n"
```

Comment:
In this case, the path `docs/assets/tach-yml-schema.json` will only work within our repo. External users typically use our hosted version: `https://gauge-sh.github.io/tach/assets/tach-yml-schema.json`

Could you move that URL into a constant and reference the constant here?

Reply:
Yes sure!
I had more more doubt is there a right place for creating the TachYamlDumper class. I found it odd being in the same file.

Reply:
Great thanks! And for the placement, I don't see an issue, since the `parsing` module is responsible for the YAML serialization/deserialization and this is the only place the dumper would be used.